src/showcase/src/pages/components/PickAnchor.astro

```diff
@@ -0,0 +1,63 @@
+---
+import "$src/styles/main.css";
+---
+
+<div>
+  <pick-anchor></pick-anchor>
+</div>
+
+<script>
+  import { asyncReplace } from "lit-html/directives/async-replace.js";
```

Comment:
the explicit `.js` in the import is mandatory?

Reply:
yep: https://lit.dev/docs/templates/directives/#asyncreplace